### DISCUSSION TITLE
NanoString Plexset Demultiplexing

### DISCUSSION THREAD

**Discussion by colsen-bu** [mcanouil/NACHO#44] [Q&A] [answered]
Hi! I am working with a NanoString PlexSet data set and I am trying to figure out the best way to demultiplex it to remove some of the samples from the normalization process which I know to contain technical errors. I am not sure about the best way to go about doing this and any insights would be greatly appreciated thanks!

**Comment by mcanouil** [comment, marked as answer]
Hi,

You could modify the returned object from `NACHO::load_rcc()`.
For example:

```r
library(NACHO)
data(GSE74821) # results from NACHO::load_rcc()
GSE74821[["nacho"]] <- dplyr::filter(
  .data = GSE74821[["nacho"]],
  <some filter of your own making>
)
normalise(GSE74821)
```

**Reply by colsen-bu** [reply]
This worked well for me, thank you so much!
